Orchestrator/docs/NLRModels.md

```diff
@@ -0,0 +1,29 @@
+## 									-- DRAFT --
+
+
+
+# Natural Language Representation Model
+
+
+Turing Natural Language Representation (Turing *NLR*) models are generic language representation models which have been trained towards more sophisticated pretraining tasks for both monolingual as well as multilingual scenarios. Turing NLR models are used as a natural replacement for BERT-like models.
+
+## Models
+
+### Turing NLR v3
```

Comment:
Let's remove the Turing reference.

Reply:
removed entire section. left a shell to be filled.